containers/ws/README.md

```diff
@@ -1,99 +1,153 @@
-# Cockpit on Fedora CoreOS or other container hosts
+# Cockpit WS Container
 
-The standard Fedora and Red Hat Enterprise Linux CoreOS images does not contain
-Cockpit packages.
+This container is suitable for scenarios where you can not or wish not to deploy the Cockpit Web Service and browser
+application natively.
 
-1. Install Cockpit packages as overlay RPMs:
+## Tags
+
+The following tags are provided for the image:
+
+- `:latest`, `:latest-amd64`, `:latest-arm64` point to the latest images and their respective architecture
+- `:{Cockpit WS Version}`, `:{Cockpit WS Version}-amd64`, `:{Cockpit WS Version}-arm64` (e. g. `:280-amd64`) point to
+  images with the specified Cockpit WS version and their respective platform
+
+## Preparing Managed Host
+
+The hosts you want to manage via this container need to have both `cockpit-system` installed and SSH access enabled.
+
+Depending on your configuration, you may want to install
+[additional extensions](https://packages.fedoraproject.org/search?query=cockpit-) as well, such as `cockpit-kdump` or
+`cockpit-networkmanager`.
+
+The standard Fedora and Red Hat Enterprise Linux CoreOS images do not provide Cockpit packages. If you wish to install
+the container on CoreOS or simply manage such a system, you need to follow these
+steps to setup Cockpit:
+
+1. Install Cockpit core packages as overlay RPMs:
    ```
    rpm-ostree install cockpit-system cockpit-ostree cockpit-podman
    ```
 
-   Depending on your configuration, you may want to use
-   [other extensions](https://apps.fedoraproject.org/packages/s/cockpit-) as
-   well, such as `cockpit-kdump` or `cockpit-networkmanager`.
-
    If you have a custom-built OSTree, simply include the same packages in your build.
 
 2. Reboot
 
-Steps 1 and 2 are enough when the CoreOS machine is only connected to through another host running Cockpit.
```

Comment:
This piece of information should stay. In fact, we actively promote to *not* install webservers willy-nilly, and prefer SSH mode whenever possible.